polynote-frontend/polynote/ui/component/notebook/cell.ts

```diff
@@ -1991,8 +2007,17 @@ class CodeCellOutput extends Disposable {
                             ]);
 
                             if (report.position) {
-                                const lineNumber = linePosAt(this.cellState.state.content, report.position.point)
-                                el.appendChild(span(['error-link'], [`(Line ${lineNumber})`]))
+                                // TODO: this is really discombobulated. Why does this have to call this function
+                                //        when the cell uses the editor model to translate the position? It's not
+                                //        "cleaner". I guess the cell should have a mapped view of the compile errors
+                                //        and pass that to the output component instead?
+                                const linePos = linePosAt(this.cellState.state.content, report.position.point)
```

Comment:
Since this is the only usage of `linePosAt` maybe we should just have it return a 1-based line number?